pkg/cloudprovider/aws/amifamily/bottlerocket.go

```diff
@@ -0,0 +1,100 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package amifamily
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/aws/karpenter/pkg/apis/provisioning/v1alpha5"
+	"github.com/aws/karpenter/pkg/cloudprovider"
+	"github.com/aws/karpenter/pkg/cloudprovider/aws/apis/v1alpha1"
+	"github.com/aws/karpenter/pkg/utils/functional"
+	"github.com/aws/karpenter/pkg/utils/injection"
+)
+
+func init() {
+	bottlerocket := Bottlrocket{}
+	amiFamilies[bottlerocket.Name()] = bottlerocket
+}
+
+type Bottlrocket struct{}
+
+// Name is the name of the AMI Family
+func (b Bottlrocket) Name() string {
+	return v1alpha1.AMIFamilyBottlerocket
```

Comment:
If we're maintaining a map, do we need to include .Name()? Are we using .Name() elsewhere?